.travis.yml

```diff
@@ -21,10 +21,12 @@ before_script:
   - ./configure --enable-ccache
 script:
   - make tidy
-  - make rustc-stage1 -j4
+  - make check -j4
 
 env:
   - CXX=/usr/bin/g++-4.7
+  matrix:
```

Comment:
Should it just not be nested under env? The documentation seems unclear (I've only ever seen matrix nested under env).